Process.c

```diff
@@ -381,6 +723,33 @@ void Process_writeField(const Process* this, RichString* str, ProcessField field
       Process_writeCommand(this, attr, baseattr, str);
       return;
    }
+   case PROC_COMM: {
+      const char* procComm;
+      if (this->procComm) {
+         attr = CRT_colors[Process_isUserlandThread(this) ? PROCESS_THREAD_COMM : PROCESS_COMM];
+         procComm = this->procComm;
+      } else {
+         attr = CRT_colors[PROCESS_SHADOW];
+         procComm = Process_isKernelThread(this) ? kthreadID : "N/A";
+      }
+      /* 15 being (TASK_COMM_LEN - 1) */
+      Process_printLeftAlignedField(str, attr, procComm, 15);
```

Comment:
maybe just use `TASK_COMM_LEN - 1` instead of `15` and remove the comment?
Every compiler should compute it at compile time.